specification/network/resource-manager/Microsoft.Network/stable/2018-07-01/loadBalancer.json

```diff
@@ -1459,7 +1459,9 @@
         }
       },
       "required": [
-        "backendAddressPool"
```

Comment:
@khannarhea @MikhailTryakhov why the changes for `2018-07-01` are targeting September branch (2018-08-01)? IMHO, it should target `master`.

Reply:
@sergey-shandar  you're right. @khannarhea  please retarget it to ```master``` branch